Project.toml

```diff
@@ -23,10 +23,10 @@ Markdown = "<0.0.1, 1"
 Random = "<0.0.1, 1"
 RecipesBase = "1"
 Reexport = "1"
-TaylorIntegration = "0.15"
-TaylorSeries = "0.17"
+TaylorIntegration = "0.16"
+TaylorSeries = "0.18.4"
```

Comment:
Did you have to go directly to `TaylorSeries = "0.18.4"` or could you also go to `TaylorSeries = "0.18"`?

Reply:
I think you need to upgrade to TS v0.18.4...